### PR TITLE
[Forta-659] Build tests for Finding

### PR DESCRIPTION
**Pull request by briankwong65**

### CODE REVIEW

**Comment by haseebrabbani** [comment]
looks like the build is still breaking?

File: sdk/network.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  ROPSTEN = 3,
+  RINKEBY = 4,
+  GOERLI = 5,
+  POLYGON = 137,
+  BSC = 56,
+  AVALANCHE = 43114,
+  ARBITRUM = 42161,
+  OPTIMISM = 10,
+  FANTOM = 250,
+}
```
Comment (Robert-H-Leonard) [inline sdk/network.ts:12]: 💚

File: sdk/finding.ts [hunk 2]
```diff
@@ -49,8 +49,6 @@ export class Finding {
       ...this,
       severity: FindingSeverity[this.severity],
       type: FindingType[this.type],
-      labels: this.labels.map(l => Object.assign(l, {
-        entityType: EntityType[l.entityType]}))
```
Comment (Robert-H-Leonard) [inline sdk/finding.ts:53]: Should this have been removed?

**Comment by briankwong65** [comment]
> looks like the build is still breaking?

Fixed, just because we missed some attributes in FortaConfig

File: sdk/finding.ts (hunk 2 above)
Comment (haseebrabbani) [reply sdk/finding.ts:53]: we should leave this as is. this is meant for printing the finding in human-readable format. this line is converting the label.entityType into a string (i believe its an integer by default)

File: sdk/transaction.event.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { EventType, Network } from "./index";
 import { Log } from "./receipt";
 import { Trace } from "./trace";
 import { Transaction } from "./transaction";
-import { ethers } from ".";
```
[the remark this answers is not in the document]
Comment (haseebrabbani) [reply sdk/transaction.event.ts:5]: see https://github.com/forta-network/forta-bot-sdk/pull/286

File: sdk/finding.ts (hunk 2 above)
Comment (briankwong65) [reply sdk/finding.ts:53]: But I think when we create the label object, label.entityType is already converted into a string:
https://github.com/forta-network/forta-bot-sdk/blob/b9d5f577dfdb945a192fe60af4a06575115bfb49/sdk/label.ts#L45
So this line shall be duplicated, thus, when I run the test with this line not deleted, it returns the index but not the string. Oppositely, When I removed this, it returns the string correctly.

Comment (haseebrabbani) [reply sdk/finding.ts:53]: the above line is converting a string value to an _enum value_. the Typescript type of the field `entityType` is an enum. but when we print it to console/logs, we want it to appear as a string so we convert it back

Comment (briankwong65) [reply sdk/finding.ts]: Ahh, I think I mixed up something, just updated the branch!

**Review by haseebrabbani — APPROVED** [review]

**Pull request merged**